findmy/accessory.py

```diff
@@ -70,6 +72,7 @@ class FindMyAccessory(RollingKeyPairSource):
 
     def __init__(  # noqa: PLR0913
         self,
+        *,
```

Comment:
This is a breaking change to make the init kwarg-only. I think this is a very good change though, the unnamed list of args is confusing otherwise.